.changeset/salty-things-refuse.md

```
@@ -0,0 +1,6 @@
+---
+'@mastra/clickhouse': minor
+'@mastra/pg': minor
+---
+
+Added support for filtering scores by metadata key-value pairs in listScores.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)